base/boot.jl

```diff
@@ -125,9 +125,10 @@ export
     Box, Function, IntrinsicFunction, LambdaStaticData, Method, MethodTable,
     Module, Symbol, Task, Array, GenSym,
     # numeric types
-    Bool, FloatingPoint, Float16, Float32, Float64, Number, Integer, Int, Int8, Int16,
-    Int32, Int64, Int128, Ref, Ptr, Real, Signed, UInt, UInt8, UInt16, UInt32,
-    UInt64, UInt128, Unsigned,
+    Real, Complex, Number, Integer, Bool, Ref, Ptr,
```

Comment:
i wonder if it should move here (or had at some points in the life of this PR), since it is now an influential type on ccall ABI generation